include/text.php

```diff
@@ -933,12 +933,12 @@ function contact_block() {
 	$a = get_app();
 
 	$shown = get_pconfig($a->profile['uid'],'system','display_friend_count');
-	if($shown === false)
+	if ($shown === false)
```

Comment:
Added them here, I have only rebased my 2-6-2 branch to make sure I have latest commits in. It is now committed. :-)